### PR TITLE
Added documentation of pyDataverse conda installation

### PR DESCRIPTION
**Pull request by PennyHow**
These are minor edits to pyDataverse docs to include documentaion on the new conda install, as discussed here - https://github.com/gdcc/pyDataverse/issues/159

**Platforms**
* [X] OS: NAME, VERSION, 64/32bit 
    * Linux 64-bit
* [X] pyDataverse: VERSION 
    * v0.3.1
* [X] Python: VERSION 
    * Python 3.8
* [ ] Dataverse: VERSION 
    * NA

**Follow best practices**
* [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/gdcc/pyDataverse/pulls) for the same update/change?
* [X] Have you followed the guidelines in our [Contribution Guide](https://pydataverse.readthedocs.io/en/master/contributing/contributing.html)?
* [X] Have you read the [Code of Conduct](https://github.com/gdcc/pyDataverse/blob/master/CODE_OF_CONDUCT.md)?
* [X] Do your changes in a seperate branch. Branches MUST have descriptive names.
* [X] Have you merged the latest changes from upstream to your branch?

**Describe the PR**
* [X] What kind of change does this PR introduce?
  * docs
* [X] Why is this change required? What problem does it solve?
  * Updating pyDataverse docs with information about how to install the package with conda
* [ ] Screenshots (if appropriate)
* [X] Put `Closes #ISSUE_NUMBER` to the end of this pull request 

**Testing**
* [ ] Have you used tox and/or pytest for testing the changes?
* [ ] Did the local testing ran successfully?
* [ ] Did the Continous Integration testing (Travis-CI) ran successfully?

**Commits**
* [X] Have descriptive commit messages with a short title (first line).
* [ ] Use the [commit message template](https://github.com/gdcc/pyDataverse/blob/master/.github/.gitmessage.txt)
* [ ] Put `Closes #ISSUE_NUMBER` in your commit messages to auto-close the issue that it fixes (if such).

**Others**
* [X] Is there anything you need from someone else?
  * If anyone else has anything to add then please do. Also, I added the conda install instructions to the `index.rst` page as well as the installation page. I know though that you might want to keep the front page simplified and only include the conda install on the installation page (along with the instructions on installing with pipenv). If so, then please feel free to alter accordingly.

### Documentation contribution
* [X] Have you followed NumPy Docstring standard?

### Code contribution
* [ ] Have you used pre-commit?
* [ ] Have you formatted your code with black prior to submission (e. g. via pre-commit)?
* [ ] Have you written new tests for your changes?
* [ ] Have you ran mypy on your changes successfully?
* [ ] Have you documented your update (Docstrings and/or Docs)?
* [ ] Do your changes require additional changes to the documentation?

Closes https://github.com/gdcc/pyDataverse/issues/159

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -485,7 +485,7 @@ many current examples of tests, and we suggest looking to these for inspiration.
 Name your tests with a descriptive filename (with prefix ``test_``) and put it
 in an appropriate place in the ``tests/`` structure.
 
-Follow the typical pattern of constructing an ``expected`` and comparing versus
+Follow the typical pattern of constructing an ``expected`` and comparing versushttps://pypi.org
```
Comment (pdurbin) [inline CONTRIBUTING.rst:488]: This seems to be a typo.

**Review by pdurbin — COMMENTED** [review]
Overall, this looks good.

I think I found a typo.

I didn't try running `make html`. Maybe someday we could set up the nice features we have for Dataverse itself:

- a check to make sure `make html` doesn't produce any errors
- nice previews of the HTML from the pull request, like this: https://dataverse-guide--9600.org.readthedocs.build/en/9600/

File: CONTRIBUTING.rst (hunk 1 above)
Comment (PennyHow) [reply CONTRIBUTING.rst]: Typo removed

**Comment by PennyHow** [comment]
> I think I found a typo.

Don't know where this came from, but I have changed it.

I also just remembered to add in the conda-forge badge to the `README.md` -  [Conda Version](https://anaconda.org/conda-forge/pydataverse)

**Comment by pdurbin** [comment]
@PennyHow hi! Sorry, sorry, but this pull request now has merge conflicts! Can you please resolve them?

It's probably because today we (ok @JR-1991) but released the first version of pyDataverse in over three years! 🎉 

This kicked off Conda activity such as this:

- https://github.com/conda-forge/pydataverse-feedstock/pull/1

(Which I must admit that I don't understand but I'm sure you and @izahn do!)

Anyway, yay pyDataverse! Yay Conda! 🎉 Thanks again for this pull request!

**Comment by PennyHow** [comment]
Hi @pdurbin I've just closed this PR and will open a new one with the merg conflicts resolved. 

And congrats on the new pyDataverse release 🎉 I will take a look at the feedstock 👍🏼

**Pull request closed without merging**